src/Search/Indexes/SearchIndex.php

```diff
@@ -565,7 +565,7 @@ protected function _getFieldValue($object, $field)
      */
     public static function warn($e)
```

Comment:
You could change the method name to `info` as well, so long as you also implement a `warn` function that marks deprecated and calls `info` :)
It would require a minor number version bump though, but I think that's probably fine.

Reply:
Good idea. I could add a new method but ideally this would be best as a patch so it can stop breaking things. Currently these errors are just dumped out regardless of context - especially annoying in `Content-Type: application/json` responses...

Reply:
Haha. I think eventually - perhaps when the default logger implementation is improved in core, there could be some work done here. Switch it back to a more appropriate level, preferably remove the random public API on this class for logging. Considering that, I'd prefer to not do any API updates, as I think the API could be improved significantly with a more directed PR.

Reply:
OK, cool :)